packages/beacon-node/src/network/core/networkCoreWorker.ts

```diff
@@ -104,6 +104,7 @@ const core = await NetworkCore.init({
     reqRespBridgeRespCaller.getAsyncIterable({method, req, peerId: peerIdToString(peerId)}),
   activeValidatorCount: workerData.activeValidatorCount,
   initialStatus: workerData.initialStatus,
+  genesisTime: workerData.genesisTime,
```

Comment:
We are passsing the `clock` object already, so where we need `genesisTime` inside `NetworkCore` we can use `clock.genesisTime` don't need to pass it as an extra attribute.